.github/publish-ci/cra5/package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "dependencies": {
     "@react-spring/web": "*",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
+    "react": "^19.1.0",
```

Comment:
NEED REVIEWER HERE

Changes:
- [ ] All dev dependencies bumped to React 19
- [ ] All demo/ci dependencies bumped to React 19
- [ ] styledComponent updated to support React 19
- [ ] update demo: @react-three/drei update to support React 19
- [ ] update demo: @react-three/fiber update to support React 19
- [ ] update demo: @react-select update to support React 19